custom/enikshay/management/commands/base_data_dump.py

```diff
@@ -49,6 +49,8 @@ def generate_dump(self):
                 # that might be needed repeatedly for the same case and is expensive call
                 self.context = {}
                 case_row = {}
+                if not self.include_case_in_dump(case):
+                    continue
```

Comment:
Looks like this is now expected to be present as a method. We should add `include_case_in_dump` method in case and default it to return `True` and then overwrite where needed?

Reply:
Good catch, yes.  Will add.